LayoutTests/http/tests/site-isolation/iframe-and-window-open.html

```diff
@@ -15,9 +15,10 @@
     if (event.data == 'initial ping') {
         openedWindow.postMessage('initial reply', '*');
         return;
+    } else if (event.data.startsWith("opened window received: ")) {
+        testPassed("received message from opened window: " + event.data);
+        finishJSTest();
```

Comment:
I needed to make this change because of the change I made to `green-background.html` (which this test uses). 
This test needs to ignore the postmessage I added from the `green-background.html` iframe.

Reply:
If the frame is expected to receive `onload` message before the `opened window` message, we should log the `onload` message and update -expected; but ignoring it is fine since it's only one message.
Let's just update the `else if` part then.